samples/snippets/create_custom_entry.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_custom_entry(override_values):
+    """Creates a custom entry within an entry group."""
+    # [START data_catalog_create_custom_entry]
+    # Import required modules.
+    from google.cloud import datacatalog_v1
+
+    # Google Cloud Platform project.
+    project_id = "my-project"
+    # Entry group to be created.
+    # For sample code demonstrating entry group creation, see quickstart.
```

Comment:
Should we just link to the sample browser URL for the quickstart?  Or just the samples root for datacatalog?  Can also revisit this after this is submitted and see if we get feedback about navigation..

Reply:
Added link to c.g.c quickstart. Will update to point to Sample Browser page once Data Catalog is onboarded.